docs/tutorial/theme-tutorials.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Theme Tutorials
+title: Plugin and Theme Tutorials
 ---
 
 Gatsby themes are a new way to encapsulate, re-use, and compose Gatsby functionality.
```

Comment:
We should update this intro to mention Plugins in addition to Themes. Perhaps you could leverage some of the comparisons here or in the docs overview sections for Plugins and Themes: https://www.gatsbyjs.org/docs/plugins-themes-and-starters/